WoWPro_Leveling/Wrath/Horde/WOTLK_Howling_Fjord.lua

```diff
@@ -107,56 +107,59 @@ A Mastering the Runes |QID|11351|PRE|11350|N|From Longrunner Pembe.|
 T March of the Giants |QID|11365|M|48.1,10.5|N|To Sage Edan.|
 A The Lodestone |QID|11366|PRE|11365|N|From Sage Edan.|
 A Skorn Must Fall! |QID|11256|PRE|11281|M|48.1,10.8|N|From Chieftain Ashtotem.|
-
-;HerniaOG Audit... so far
-
 R Skorn |QID|11258|N|Run to Skorn.|M|44,26|
-T Skorn Must Fall! |QID|11256|U|33340|N|Use the Winterhoof Emblem and speak to the Winterhoof Brave.|
-A Stop the Ascension!|QID|11260|U|33345|N|From UI Alert.|O|
+T Skorn Must Fall! |QID|11256|U|33340|M|44.33,26.27|N|Use the Winterhoof Emblem and speak to the Winterhoof Brave.|
 A Gruesome, But Necessary |QID|11257|PRE|11256|N|From the Winterhoof Brave.|
 A Burn Skorn, Burn! |QID|11258|PRE|11256|N|From the Winterhoof Brave.|
 A Towers of Certain Doom |QID|11259|PRE|11256|N|From the Winterhoof Brave.|
-;l Vrykul Scroll of Ascension |QID|11260|S|L|33345|N|Until you get the Vrykul Scroll of Ascension.|
-C Gruesome, But Necessary |QID|11257|U|33342|S|N|When you kill a Vrykul, use The Brave's Machete on the corpse.|
+l Vrykul Scroll of Ascension |AVAILABLE|11260|S|L|33345|M|44.33,26.27|N|Until you get the Vrykul Scroll of Ascension.|
+C Gruesome, But Necessary |QID|11257|U|33342|S|M|44.33,26.27|N|When you kill a Vrykul, use The Brave's Machete on the corpse.|
 N Burn the Northwest Longhouse |QID|11258|QO|1|U|33343|N|Just to the southwest. Go inside and use the Brave's Torch.|M|43.7,28.6|
```

Comment:
@EzeRage All these N steps should be changed to C steps with a QO tag.

Reply:
The N steps were there when the guide was originally made. So do I just change them to C?

Reply:
yes. 

C Target the Northwest Tower |QID|11259|QO|1|U|33344|N|It's just south. Don't get too close or the mages on it will attack you. The Brave's Flare has a 90 yard range.|M|44.4,28.6|